### ISSUE TITLE
Animate card on button press [Feature] 

### ISSUE THREAD

**Issue by bizlal** [JoniVR/VerticalCardSwiper#61]
It would be cool if we could animate the card off the screen based on a button press. If anyone has done this already, please help. 

This would be kind of like scrollToCard. 
### New Issue Checklist

- [x] I read the [documentation](https://github.com/JoniVR/VerticalCardSwiper).
- [x] I searched for [existing GitHub issues](https://github.com/JoniVR/VerticalCardSwiper/issues).

**Comment by JoniVR** [comment]
Forgot to reference the issue in the commit, it's been implemented in 1ee6c5b :)
I'll try to release a new version soon but I was actually planning on testing some more stuff before that. 

If you want to add it right now you could modify your podfile line to something like: 
```sh
pod 'VerticalCardSwiper', :git => 'https://github.com/JoniVR/VerticalCardSwiper.git', :branch => 'master'
```

**Comment by JoniVR** [comment]
Version `2.1.0` has been released which includes the `swipeCardAwayProgrammatically` function! 🎉 

Closing this now 😄 let me know if you happen to run into any issues with it.
